packages/eslint-plugin-svelte/tests/fixtures/rules/derived-has-same-inputs-outputs/invalid/test01-errors.yaml

```diff
@@ -1,32 +1,1044 @@
+  line: 46
+  column: 19
+  suggestions:
+    - desc: Rename the parameter from g to $e.
+      messageId: renameParam
+      output: |
+        import { derived } from 'svelte/store';
+
+        derived(a, (b) => {
+        	/** do nothing */
+        });
+        derived(c, (d, set) => {
+        	/** do nothing */
+        });
+        derived([e, f], ([g, h]) => {
+        	/** do nothing */
+        });
+        derived([i, j], ([k, l], set) => {
+        	/** do nothing */
+        });
+        derived([null, l], ([$m, $n]) => {
+        	/** do nothing */
+        });
+        derived([o, null], ([$p, $q]) => {
+        	/** do nothing */
+        });
+        derived(a, (b) => {
+        	doSomethingWith(b);
+
+        	somethingWithACallback(() => {
+        		b;
+        	});
+        });
+        derived(a, (b) => {
+        	b;
+
+        	somethingWithACallback(() => {
+        		// purposely shadow the var, this should not be updated
+        		const b = 303;
+        		b;
+        	});
+        });
+        derived([e, f], ([g, h]) => {
+        	g;
+        	h;
+
+        	somethingWithACallback(() => {
+        		g;
+        		h;
+        	});
+        });
+        derived([e, f], ([$e, h]) => {
+        	$e;
+        	h;
+
+        	somethingWithACallback(() => {
+        		const g = 303;
+        		const h = 808;
+        		g;
+        		h;
+        	});
+        });
+        derived(a, (b) => {
+        	// cause a conflict in names so the suggestion can't work
+        	const $a = 303;
+        	$a;
+        });
+        somethingWithACallback(() => {
+        	const $a = 303;
+        	derived(a, (b) => {
+        		$a;
+        	});
+        });
+- message: The argument name should be '$f'.
+  line: 46
+  column: 22
+  suggestions:
+    - desc: Rename the parameter from h to $f.
+      messageId: renameParam
+      output: |
+        import { derived } from 'svelte/store';
+
+        derived(a, (b) => {
+        	/** do nothing */
+        });
+        derived(c, (d, set) => {
+        	/** do nothing */
+        });
+        derived([e, f], ([g, h]) => {
+        	/** do nothing */
+        });
+        derived([i, j], ([k, l], set) => {
+        	/** do nothing */
+        });
+        derived([null, l], ([$m, $n]) => {
+        	/** do nothing */
+        });
+        derived([o, null], ([$p, $q]) => {
+        	/** do nothing */
+        });
+        derived(a, (b) => {
+        	doSomethingWith(b);
+
+        	somethingWithACallback(() => {
+        		b;
+        	});
+        });
+        derived(a, (b) => {
+        	b;
+
+        	somethingWithACallback(() => {
+        		// purposely shadow the var, this should not be updated
+        		const b = 303;
+        		b;
+        	});
+        });
+        derived([e, f], ([g, h]) => {
+        	g;
+        	h;
+
+        	somethingWithACallback(() => {
+        		g;
+        		h;
+        	});
+        });
+        derived([e, f], ([g, $f]) => {
```

Comment:
It should be

```suggestion
        derived([e, f], ([$e, $f]) => {
```

Reply:
it shouldn't as the suggestions apply one at a time. this is listing the output of each individual suggestion